includes/class-wc-payments-order-success-page.php

```diff
@@ -73,6 +113,35 @@ public function show_woopay_payment_method_name( $payment_method_title, $abstrac
 		return ob_get_clean();
 	}
 
+	/**
+	 * Add the BNPL logo to the payment method name on the order received page.
+	 *
+	 * @param WC_Payment_Gateway_WCPay $gateway the gateway being shown.
+	 * @param WCPay\Payment_Methods\UPE_Payment_Method $payment_method the payment method being shown.
+	 *
+	 * @return string|false
+	 */
+	public function show_bnpl_payment_method_name( $gateway, $payment_method ) {
+		$method_logo_url = apply_filters(
+			'wc_payments_thank_you_page_bnpl_payment_method_logo_url',
+			$payment_method->get_payment_method_icon_for_location( 'checkout', false, $gateway->get_account_country() ),
+			$payment_method->get_id()
+		);
+
+		// If we don't have a logo URL here for some reason, bail.
+		if ( ! $method_logo_url ) {
+			return false;
+		}
+
+		ob_start();
+		?>
+		<div class="wc-payment-gateway-method-logo-wrapper wc-payment-bnpl-logo <?php echo str_replace( 'woocommerce_payments_', '', $payment_method->get_id() ); ?>">
```

Comment:
```suggestion
		<div class="wc-payment-gateway-method-logo-wrapper wc-payment-bnpl-logo <?php echo $payment_method->get_id(); ?>">
```
If I'm following along correctly, `get_id()` on the `UPE_Payment_Method` class will return the [Stripe ID](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/payment-methods/class-upe-payment-method.php#L116), not the payment gateway ID, so I think removing this prefix is redundant.

Reply:
Yep. You're correct. I think this was left over from a previous iteration where I was using the `payment_method_id` directly from the order and not the payment method.